apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -58,9 +58,11 @@ function hasPendingOps(account: Account) {
   );
 }
 
-async function maybeSync(fromScratch?: boolean) {
+type SyncStatus = "success" | "failed" | "skipped" | "skipped";
+
+async function maybeSync(fromScratch?: boolean): Promise<SyncStatus> {
   const manager = getAccountManager();
-  if (manager.currentAccount == null) return;
+  if (manager.currentAccount == null) return "skipped";
```

Comment:
@wojtus7 there was a bug here during onboarding due to the splash screen change: #605